packages/inter-protocol/src/proposals/upgrade-vaults.js

```diff
@@ -131,9 +130,10 @@ export const upgradeVaults = async (
       E.get(reserveKit).creatorFacet,
     ).makeShortfallReportingInvitation();
 
+    // we want the auctioneer instance after auctionsUpgradeComplete settles
     const [poserInvitation, auctioneerInstance] = await Promise.all([
       E(electorateCreatorFacet).getPoserInvitation(),
-      auctioneerInstanceP,
+      E.get(consumeInstance).auctioneer,
```

Comment:
Interesting approach.

This PromiseSpace synchronization technique is clearly "too clever by half", since I didn't catch that in code review earlier.

I did start to suggest checking that agoricNames got updated, but only for the installation. And just checking that the instance changed in agoricNames wouldn't tell us which instance the vaultFactory picked up.